src/yamlfix/entrypoints/__init__.py

```diff
@@ -6,38 +6,49 @@
 
 import logging
 import sys
+from enum import Enum
 
-# Ansi color codes
-RED = 31
-YELLOW = 33
-CYAN = 36
-GREEN = 32
+
+class ANSIFGColorCode(Enum):
+    """ANSI escape codes for colored output."""
+
+    BLACK = 30
+    RED = 31
+    GREEN = 32
+    YELLOW = 33
+    BLUE = 34
+    MAGENTA = 35
+    CYAN = 36
+    WHITE = 37
+    RESET = 0
 
 
 class ConsoleColorFormatter(logging.Formatter):
     """Custom formatter that prints log levels to the console as colored plus signs."""
 
+    # ANSI escape codes for colored output
     colors = {
-        logging.DEBUG: GREEN,
-        logging.INFO: CYAN,
-        logging.WARNING: YELLOW,
-        logging.ERROR: RED,
+        logging.DEBUG: ANSIFGColorCode.GREEN,
+        15: ANSIFGColorCode.BLUE,
```

Comment:
15 is half-way between `DEBUG` and `INFO`.
I feel that `DEBUG` is too low for printing unchanged files, since it contains a lot of spam messages.
I see two possible solutions:
* Move all current debug logging down to `NOTSET/TRACE` and log unchanged files at debug.
* Move normal logging to `WARNING` level and log unchanged files at `INFO`.

Reply:
I see what you mean. Moving `log.warn` can be misleading as the content of the message is not a warning. I think the cleaner way is to move the current `DEBUG` to `NOTSET/TRACE`

Reply:
@lyz-code So, turns out `NOTSET` is not a real level in python logging, it doesn't have its own function and you can't log at it. So we need to use numbered level. IMHO having one instance of `log.log(15, ...)` is much better than having `log.log(1, ...)` everywhere. So I kept it as is for now.

Reply:
Damn, ok. Can you please add a comment explaining all this on the definition of the log level `15`? I want to avoid the WTF moment in 6 months when I read those lines

Reply:
Done